### PR TITLE
Fix seg_fault when doing setup_deb.sh

### PR DESCRIPTION
**Pull request by frank-onspecta**
We keep on having seg fault in calling setup_deb.sh in fresh installation. The issue is that tensorflow package needs to be imported prior to pytorch. Otherwise issues like this will keep on recurring

https://jenkinsai.amperecomputing.com/job/Releases/job/tensorflow-dls/17/console

### CODE REVIEW

File: setup_deb.sh [hunk 1]
```diff
@@ -98,7 +98,16 @@ pip3 install --no-deps --upgrade \
...
+   tokenizers==0.12.1 \
+   absl-py \
+   wrapt \
+   opt_einsum \
+   gast \
+   astunparse \
+   termcolor \
+   keras_preprocessing \
+   flatbuffers \
+   tensorflow==2.7.1
```
Comment (jan-grzybek-ampere) [inline setup_deb.sh:110]: Installing tensorflow here will override tensorflow-aio present in e.g. our docker or some dev environment with native version.

Comment (jan-grzybek-ampere) [reply setup_deb.sh:110]: Thats why there is sort of a hack with install_frameworks.py script

Comment (jan-grzybek-ampere) [reply setup_deb.sh:110]: @dkupnicki would you please have a look

**Review by jan-grzybek-ampere — CHANGES_REQUESTED** [review]

File: setup_deb.sh (hunk 1 above)
Comment (frank-onspecta) [reply setup_deb.sh]: @jan-grzybek-ampere , you are correct. The pip install tensorflow should be removed. Actually, I think the simplest fix is simply moving the TF check in install_frameworks.py to be the first one.

Comment (frank-onspecta) [reply setup_deb.sh]: @dkupnicki , please investigate the CI test issue with the bert model.

**Comment by jan-grzybek-ampere** [comment]
@dkupnicki is your fix on https://github.com/AmpereComputingAI/ampere_model_library/pull/160 addressing the very same issue?

**Comment by dkupnicki** [comment]
#160 addresses CI failing on this PR (and other PRs), but this PR fixes a completely different issue

**Review by dkupnicki — APPROVED** [review]
Looks good

**Review by jan-grzybek-ampere — APPROVED** [review]

**Pull request merged**